### PR TITLE
Added a new logo

### PR DESCRIPTION
**Pull request by francylang**
@ameseee @nicktu12 @robbiegreiner 
## What's this PR do?
Changed up the logo.

## Where should the reviewer start?
Just let me know what you think!
See screenshots below

## Screenshots
BEFORE:
image

AFTER:
image

### CODE REVIEW

File: lib/components/Classroom/Classroom.js [hunk 1]
```diff
@@ -6,15 +6,14 @@ import { Redirect } from 'react-router';
...
-  }
+  // if (userStatus === false) {
+  //   return <Redirect to='/login'/>;
+  // }
 
   return (
     <div>
-      <div className='class-drop-down'>
         <h2 className='teacher-name'>{teacher}'s Classroom</h2>
-      </div>
```
Comment (nicktu12) [inline lib/components/Classroom/Classroom.js]: this is scary

Comment (francylang) [reply lib/components/Classroom/Classroom.js]: Sometimes I write my code in blood.

**Comment by ameseee** [comment]
Can I get the contact info for your graphic designer?

**Pull request closed without merging**